GitVersionCore/VersionCalculation/FallbackBaseVersionStrategy.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace GitVersion.VersionCalculation
+{
+    using System.Linq;
+    using GitVersion.VersionCalculation.BaseVersionCalculators;
+
+    public class FallbackBaseVersionStrategy : BaseVersionStrategy
```

Comment:
This one should probably be in the BaseVersionCalculators folder next to the other version strategies?